src/components/reachout.module.css

```diff
@@ -1,14 +1,107 @@
-.reachout {
-  background-image: linear-gradient(var(--purple-color), var(--green-color));
+.reach-out {
+  background-color: var(--green-color);
   width: 100%;
   height: auto;
   display: flex;
   justify-content: center;
+  position: inherit;
+  z-index: -1;
 }
 
-.reachoutcontainer {
+.reach-out-container {
   transform: skew(-10deg, 5deg);
   background-color: var(--purple-color);
   width: 80vw;
-  z-index: 1;
+  z-index: 0;
+  font-family: var(--condensedblack-family);
+
+  .reach-out-info {
+    transform: skew(10deg, -5deg);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    h2 {
```

Comment:
Do all of these elements inside `.reach-out-info` need to be nested? That seems like a lot of specificity being added. Also, instead of targeting elements (like h2, form, etc), it's better to style with classes. It's more durable b/c the element can change and you want css to target presentation, not the meaning the element is conveying.